Cargo.toml

```diff
@@ -22,6 +22,7 @@ members = [
   "libraries/merkle-tree-reference",
   "memo/program",
   "name-service/program",
+  "permissioned-token/program",
```

Comment:
not too sure what you mean here. can you check that this is done correctly on the most recent update?

Reply:
Essentially, you can remove `managed-token/program/Cargo.lock`, since you've included it in the top-level workspace.  Typically you do one or the other.